prov/gni/src/gnix_ep.c

```diff
@@ -321,11 +321,15 @@ ssize_t gnix_ep_msg_injectdata(struct fid_ep *ep, const void *buf, size_t len,
  * EP RMA API function implementations.
  ******************************************************************************/
 
+#define GNIX_RMA_READ_FLAGS_DEF		(FI_RMA | FI_READ)
+#define GNIX_RMA_WRITE_FLAGS_DEF	(FI_RMA | FI_WRITE)
+
```

Comment:
These are the default completion event flags for the two RMA types.  I'm not sure we were setting RMA flags in completions correctly before.  I now have better criterion tests for this.